src/flow-worker.c

```diff
@@ -119,7 +119,7 @@ static int FlowFinish(ThreadVars *tv, Flow *f, FlowWorkerThreadData *fw, void *d
     const TcpSession *ssn = (TcpSession *)f->protoctx;
 
     /* insert a pseudo packet in the toserver direction */
-    if (client == STREAM_HAS_UNPROCESSED_SEGMENTS_NEED_ONLY_DETECTION) {
+    if (client >= STREAM_HAS_UNPROCESSED_SEGMENTS_NEED_ONLY_DETECTION) {
         Packet *p = FlowPseudoPacketGet(0, f, ssn);
         if (p != NULL) {
             PKT_SET_SRC(p, PKT_SRC_FFR);
```

Comment:
What is `FFR` stand for? This leads to a description of `stream (flow timeout)`.

Reply:
FFR: Force Flow Reassembly

A `stream (flow timeout)` AFAIU indicates that this flow needed an intervention from Suricata and was forcefully timed out bc it missed stream segments..